fdbserver/DataDistribution.actor.cpp

```diff
@@ -4329,9 +4527,11 @@ ACTOR Future<Void> dataDistribution(Reference<DataDistributorData> self)
 				teamCollectionsPtrs.push_back(remoteTeamCollection.getPtr());
 				remoteTeamCollection->teamCollections = teamCollectionsPtrs;
 				actors.push_back( reportErrorsExcept( dataDistributionTeamCollection( remoteTeamCollection, initData, tcis[1], self->dbInfo ), "DDTeamCollectionSecondary", self->ddId, &normalDDQueueErrors() ) );
+				actors.push_back(printSnapshotTeamsInfo(remoteTeamCollection, "DDTeamCollectionSecondary"));
```

Comment:
we don't need the second param `DDTeamCollectionSecondary `.

The `DDTeamCollection` struct has a field  `primary` to indicate that.
The canonical way for DD to print the primary info is to add a field `detail("Primary", self->primary)`.